### PR TITLE
Replace TravisCI with Github Action Workflow

### PR DESCRIPTION
**Pull request by bensheldon**
TravisCI has gone through a business transition and I think Github Actions, which is free for public repos, is a good alternative.

Note: there is additional contributor documentation that will need to be updated, but I would like to ensure that Github Actions is a viable alternative before doing a broad sweep of process change updates.

### CODE REVIEW

**Comment by bensheldon** [comment]
@assaf @phillbaker would one of you be able to enable Github Actions on this repository, or to provide me admin permissions so that I can enable it? Thank you 🙏

**Comment by phillbaker** [comment]
I don't have admin access - I think I reached him on twitter actually laster time: https://twitter.com/assaf

**Comment by assaf** [comment]
It says that all actions are enabled for this repository. I don't see any permission levels, only add/remove contributors.

File: .github/worklows/test.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+name: Test
+on:
+  push:
+    branches:
+      - master
+  pull_request:
```
Comment (frostmark) [inline .github/worklows/test.yml:6]: Try to add this
```suggestion
  pull_request:
    branches: '*'
```

Comment (bensheldon) [reply .github/worklows/test.yml:6]: Nuts, that wasn't the problem.

**Comment by bensheldon** [comment]
@frostmark I'm going to merge this to the main branch. I think it's _possible_ that GH actions won't kick off until there is a workflow on the main branch.

**Pull request merged**